src/App.jsx

```diff
@@ -14,33 +20,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPrepearedGood(goods, { sortField, isReversed }) {
```

Comment:
There is a typo in the function name `getPrepearedGood`. It should be `getPreparedGoods` to reflect the correct spelling of 'prepared'.